etcdadm/etcdadm

```diff
@@ -6,72 +6,70 @@ unset ETCDCTL_ENDPOINT
 
 export | grep ETCDCTL
 
+set -o nounset
+set -o errexit
+set -o pipefail
```

Comment:
Good to know! Thanks.
Are there any better ones than the current combination of options?